src/modules/LoggedIn/Project/ProjectFilter.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ProjectForm from "./ProjectForm";
+import { Row, Input, Select, Form } from "antd";
+const { Option } = Select;
+const { Search } = Input;
+const ProjectFilter = () => {
+  return (
+    <Row>
+      <Form>
+        <label style={{ marginRight: 10 }}>Sort By</label>
+        <Select defaultValue="newest" style={{ width: 120, marginRight: 10 }}>
+          <Option value="newest">Newest</Option>
+          <Option value="oldest">Oldest</Option>
+          <Option value="name">Name</Option>
+        </Select>
+        <Search
+          placeholder="Search Project Name Card"
+          style={{ width: 200, marginRight: 10 }}
+        />
+      </Form>
+      <>
+        <ProjectForm />
+      </>
+    </Row>
```

Comment:
Kalau mau bikin jarak antar flex item, lebih rapi pake props `gutter` yang ada di Row. Coba kamu baca dokumentasinya si Row https://ant.design/components/grid/#Row. Terus ProjectForm jangan ditaruh disini

```suggestion
    <Row gutter={16}>
      <Col>
        <label>Sort By</label>
      </Col>
      <Col>
       <Select defaultValue="newest" style={{ width: 120 }}>
          <Option value="newest">Newest</Option>
          <Option value="oldest">Oldest</Option>
          <Option value="name">Name</Option>
        </Select>
      </Col>
      <Col>
        <Search
          placeholder="Search Project Name Card"
          style={{ width: 200 }}
        />
      </Col>
    </Row>
```